composer.json

```diff
@@ -34,7 +34,8 @@
     },
     "autoload-dev": {
         "classmap": [
-            "tests/_fixture/"
+            "tests/_fixture/",
+            "tests/_files/"
```

Comment:
After [this](https://github.com/sebastianbergmann/global-state/pull/26/files#r321970376) has been resolved, this change is no longer needed.